src/mrinufft/density/utils.py

```diff
@@ -1,4 +1,5 @@
 """Utilities for density compensation."""
+
```

Comment:
Lets not add unnecessary spaces in history to mess things up? This can be a separate PR if it is docs or codestyle related right?

Reply:
Let me try some git wizardry to handle that.